packages/eslint-plugin-twenty-ts/src/rules/matching-state-variable.ts

```diff
@@ -0,0 +1,100 @@
+import { Rule } from 'eslint';
```

Comment:
could you use the same format as effect-component.ts to define these rules?

Reply:
Being consistent across the codebase is one of our focus :)

Reply:
OKK

Reply:
@charlesBochet Done update the effect-component.ts and the test.

Reply:
@anikdhabal i think we need to follow the format from effect-component.ts, but not the other way around

Reply:
yes exactly :) Also we don't use export default in the project :)

Reply:
fixed